### ISSUE TITLE
server started hook sent much too early

### ISSUE THREAD

**Issue by FelixReuthlinger** [lloesche/valheim-server-docker#363]
I found it often quick too early that the server started status hook gets applied.
For example here the server wasn't even downloaded, but the started hook sent signal to discord:

valheim_1  | Jun  2 10:10:51 /supervisord: valheim-server DEBUG - [157] - Valheim Server is not yet downloaded - waiting
valheim_1  | Jun  2 10:10:58 /supervisord: valheim-server DEBUG - [157] - Valheim Server is not yet downloaded - waiting
valheim_1  | 2021-06-02 10:11:01,951 INFO success: valheim-server entered RUNNING state, process has stayed up for > than 10 seconds (startsecs)
valheim_1  | Jun  2 10:11:01 /supervisord: valheim-updater valheim-server: started
valheim_1  | Jun  2 10:11:02 /supervisord: valheim-updater INFO - Running post start hook: curl -sfSL -X POST -H "Content-Type: application/json" -d "{\"username\":\"$SERVER_NAME\",\"content\":\"$DISCORD_MESSAGE_STARTED\"}" "$DISCORD_WEBHOOK"
valheim_1  | Jun  2 10:11:05 /supervisord: valheim-server DEBUG - [157] - Valheim Server is not yet downloaded - waiting
valheim_1  | Jun  2 10:11:12 /supervisord: valheim-server DEBUG - [157] - Valheim Server is not yet downloaded - waiting

**Comment by lloesche** [comment]
Can you post the entire log start to finish? There's something else going on here. The message "Valheim Server is not yet downloaded - waiting" should never be shown in the logs. If it gets into that state it means something else is broken.

**Comment by FelixReuthlinger** [comment]
This one is related to the other I posted: https://github.com/lloesche/valheim-server-docker/issues/364

So, it is basically waiting to actually download the server, but the hook already reports that the server should be running. 
Over the past month using the image, I found several cases, where this message might be a bit confusing, since the server isn't really up and running to a state, where it would really accept connections.

I don't know if there would be a better implementation for this, but maybe checking query port 2457 for response and then publish the hook message?
